doc/security/use_cases/index.md

```diff
@@ -0,0 +1,185 @@
+* Ownership Transfer: Ownership is assigned to a user that has the ability to run software on the device.
+  As Silicon Owner, the user can generate a cryptographic identity strongly associated to the hardware and the software version running on the device.
+
+OpenTitan used as a Platform Integrity Module has the following provisioning requirements:
+
+* Unique Global Identifier: Non-Cryptographic big integer value (up to 256b) used to facilitate tracking of the devices throughout their life cycle.
+  The identifier is stored in One Time Programmable (OTP) storage during manufacturing.
+* Hardware Cryptographic Identity: Symmetric and asymmetric keys associated with the hardware, used to attest the authenticity of the chip and also as a component of the Owner Cryptographic Identity.
+   These keys are generated inside the device by the secure manufacturing process.
+* Hardware Transport Certificate: Used to endorse the asymmetric hardware identity with a transit PKI trusted by the Silicon Owner at Ownership Transfer time.
+* Factory Firmware: Baseline image with support for firmware update and Ownership Transfer.
+  Firmware update may be actuated by writing an OpenTitan update payload to boot flash.
+  Upon reset, OpenTitan scans the boot flash device for valid updates.
+  The factory image may not be owned by the Silicon Owner and its main purpose is to assist Ownership Transfer.
+* Owner Cryptographic Identity: The Silicon Owner is required to generate an identity as part of the Ownership transfer flow.
+  Owner identities are bound to the Hardware Identity and the software version running on the device.
+  Owner identities are used in Silicon Ownership attestation flows and as a root component of Application keys.
+* Application Keys: Keys bound to the owner identity and the application version running on the device.
+  Application keys are provisioned in most cases when the application runs for the first time.
+  The purpose of each key is configured at the application layer and enforced by the kernel.
+
+### Performance Requirements
+
+Performance requirements are derived from integration-specific requirements.
+The following performance requirements are presented for illustration purposes:
+
+* Boot Time: Maximum time allowed to get to device kernel serving state from cold reset.
+* Resume Time: Maximum time allowed to get to device kernel serving state from sleep.
+* External Flash Verification Time: Maximum time allowed for verification of external flash as part of platform boot verified boot implementation.
+  Defined in milliseconds for a given flash partition size.
+
+### Packaging Constraints
+
+* Non-HDI packaging is required.
+* (Proposed) Device packaging QR code with device ID linkable to manufacturing data.
+
+### Additional Requirements
+
+#### Memory Requirements
+
+* At least 512KB of flash storage with 2 partitions, 4KB page size, 100K endurance cycles.
```

Comment:
not trying to throw a bomb, but we might want to agree among ourselves on the great kB vs KB (vs KiB) debate.

Reply:
Or even more fun bomb: personalization vs personalisation :)

Reply:
the bomb would say: 51 uses of kB in our repo, 21 of KB